aiven_db_migrate/migrate/pgmigrate.py

```diff
@@ -762,7 +761,7 @@ def filter_tables(self, db: PGDatabase) -> List[str]:
             return []
         if not db.tables:
             return []
-        ret = set()
+        ret: List[PGTable] = []
```

Comment:
Why change this to list? Previous implementation of using set would perform better when there's large number of tables.

Reply:
set uses `__hash__` for inclusion detection, while List will use `__eq__`. While i did implement both of them for the `PGTable` class, I figured this would be the safer choice

Reply:
Yeah it's just that the loop is O(N^2) now. Probably there aren't enough elements for that to really matter.

Reply:
I got confused on set inclusion method used (thought it was hash , it was eq). This should remain a set